Apps documentation reconstruction branch (#1226)
YunoHost/doc: app_bitwarden.md

# <img src="/images/bitwarden_logo.png" width="80px" alt="Bitwarden's logo"> Bitwarden

[![Install Bitwarden with YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=bitwarden) [![Integration level](https://dash.yunohost.org/integration/bitwarden.svg)](https://dash.yunohost.org/appci/app/bitwarden)

- [Configuration](#Configuration)
- [Limitations with YunoHost](#limitations-with-yunohost)
- [Customer Applications](#Customer-applications)
- [Useful links](#useful-links)

> Bitwarden is a freemium and open source password manager under AGPL license, which allows to generate and store passwords in a secure way. These are protected by a single password called the "master password". It is created in 2016 by Kyle Spearrin, a software architect.

Reviewer: ```suggestion
Bitwarden is a freemium libre password manager under AGPL license, allowing generation and storage of passwords in a secure way. These are protected by a single password called the "master password". It was created in 2016 by Kyle Spearrin, a software architect.
```

> The software is available on most operating systems (Linux, Windows, MacOS, iOS, Android and command line), and as a web browser plug-in. It is also possible to view passwords from a website[¹].

Reviewer: ```suggestion
The software is available on most operating systems (Linux, Windows, macOS, iOS, Android and command line), and as a web browser plug-in. It is also possible to view passwords from a website.[¹]
```

## Configuration

> To configure the application you have to go to the address: `sub.domain.tld/admin`

Reviewer: ```suggestion
To configure the app, go to this address: `sub.domain.tld/admin`
```
